archivessnake/edit_notes.py

```diff
@@ -20,15 +20,15 @@
 
 NOTE_TYPE_CHOICES = ["bioghist", "accessrestrict", "odd", "abstract", "arrangement", "userestrict", "fileplan", "acqinfo", "langmaterial", "physdesc", "prefercite", "processinfo", "relatedmaterial", "separatedmaterial"]
 ACTION_CHOICES = ["modify", "delete"]
-LEVEL = ["collection", "file", "series", "item"]
+LEVEL = ["collection", "file", "series", "item", "all"]
 CONFIDENCE_RATIO = 97
 
 def process_tree(args, resource):
     """Iterates through a given collection, file, or series provided by user input. Finds note content that matches user input and then deletes or modifies relevant notes according to user preference.  """
     for record in resource.tree.walk:
         updated = False
         aojson = record.json()
-        if record.level == args.level:
+        if record.level == args.level or args.level == "all":
```

Comment:
I believe you can do
```suggestion
        if record.level in [args.level, "all"]:
```

Reply:
I changed it to `if args.level in [record.level, "all"]:` and that worked!